lib/rubygems/commands/rdoc_command.rb

```diff
@@ -87,7 +87,7 @@ def execute
       begin
         doc.generate
       rescue Errno::ENOENT => e
-        match = / - /.match(e.message)
+        match = e.message.include?(" - ")
         alert_error "Unable to document #{spec.full_name}, " \
                     " #{match.post_match} is missing, skipping"
```

Comment:
I think this will now raise a no method error

Reply:
Dealing with this in #7392.